docs/references.rst

```diff
@@ -4,19 +4,56 @@ Ubuntu Pro Client reference
 ***************************
 
 Our reference section contains technical information for the Ubuntu Pro Client.
-This includes details of the network requirements, API definitions, and other
-related tools.
 
-Reference
-=========
+System and other requirements
+=============================
+
+The Ubuntu Pro Client is lightweight and has very few requirements. 
+
+* :ref:`Network requirements <network_requirements>`
+
+Interacting with the Client
+===========================
+
+There are two main ways to interact with the Pro Client: on the command line
+interface (CLI) in your terminal, or automatically via the API.
+
+* :ref:`CLI reference guide <commands>`
+* :ref:`API reference guide <api>`
+
+You may, in a production environment, also wish to test upcoming changes. For
+this, you can use the available Personal Package Archives (PPA) for the Ubuntu
+Pro Client.
+
+* :ref:`Personal Package Archives (PPAs) <PPAs>`
+
+You may occasionally encounter an error message when interacting with the
+Pro Client. If you do, this page will help you to resolve the error.
```

Comment:
This page only has a couple errors that are still possible to encounter, but anyone who has upgraded in the past year or so should never see them. We may need to consider removing this page.
```suggestion
You may occasionally encounter an error message when interacting with the
Pro Client. If you do, check if this page has your specific error.
```